docs/BreakingChanges/041 ObjectContext.Translate and ObjectContext.ExecuteStoreQuery supports enum.md

```diff
@@ -0,0 +1,25 @@
+## 41: ObjectContext.Translate and ObjectContext.ExecuteStoreQuery methods
```

Comment:
{
  "Id": "41",
  "Title": "ObjectContext.Translate and ObjectContext.ExecuteStoreQuery methods",
  "Details": "In .NET 4.0, the generic parameter `T` of `ObjectContext.Translate` and `ObjectContext.ExecuteStoreQuery` methods could not be an enum. That scenario is now supported.",
  "Suggestion": "If Translate or ExecuteStoreQuery was called on an enum type in .NET 4.0, '0' was returned. If that behavior was desirable, the calls should be replaced with a constant 0 (or the enum equivalent of it).",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#sql",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Data.Objects.ObjectContext.Translate`1(System.Data.Common.DbDataReader)",
    "M:System.Data.Objects.ObjectContext.Translate`1(System.Data.Common.DbDataReader,System.String,System.Data.Objects.MergeOption)",
    "M:System.Data.Objects.ObjectContext.ExecuteStoreQuery`1(System.String,System.Object[])",
    "M:System.Data.Objects.ObjectContext.ExecuteStoreQuery`1(System.String,System.String,System.Data.Objects.MergeOption,System.Object[])"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] M:System.Data.Objects.ObjectContext.Translate`1(System.Data.Common.DbDataReader)
- [ ] M:System.Data.Objects.ObjectContext.Translate`1(System.Data.Common.DbDataReader,System.String,System.Data.Objects.MergeOption)
- [ ] M:System.Data.Objects.ObjectContext.ExecuteStoreQuery`1(System.String,System.Object[])
- [ ] M:System.Data.Objects.ObjectContext.ExecuteStoreQuery`1(System.String,System.String,System.Data.Objects.MergeOption,System.Object[])